install_config/install/advanced_install.adoc

```diff
@@ -355,10 +271,125 @@ configuration.
 |`*openshift_hosted_metrics_public_url*`
 |This variable sets the host name for integration with the metrics console. The
 default is
-`*https://hawkular-metrics.{{openshift_master_default_subdomain}}/hawkular/metrics*`
+`\https://hawkular-metrics.{{openshift_master_default_subdomain}}/hawkular/metrics`
 If you alter this variable, ensure the host name is accessible via your router.
 |===
 
+[[advanced-install-deployment-types]]
+=== Configuring Deployment Type
+
+Various defaults used throughout the playbooks and roles used by the installer are
+set based on the deployment type configuration (usually defined in an Ansible
+inventory file).
```

Comment:
Remove "set" here? Seems awkward currently.
Various defaults used throughout the playbooks and roles used by the installer are based on....